spec/unit/modify_game_state_spec.rb

```diff
@@ -3,7 +3,7 @@
  require 'test_doubles/game_state_storage_gateway_fake'
 
 describe ModifyGameState do
```

Comment:
Lots of whitespace make rubocop angry

Reply:
Will use rubocop to fix up the project in a commit outside this pull request